app/src/features/surveys/sampling-information/techniques/edit/EditTechniquePage.tsx

```diff
@@ -71,6 +72,7 @@ export const EditTechniquePage = () => {
     description: technique?.description ?? null,
     distance_threshold: technique?.distance_threshold ?? null,
     method_lookup_id: technique?.method_lookup_id ?? null,
+    method_response_metric_id: 1, // NICK TODO: This is a placeholder value.
```

Comment:
nit: Still needed? Can set to `-1` to prevent issues later.

Reply:
Will check with @mauberti-bc. The technique table has a non-null foreign key to the method_response_metric table, but it appears the UI no longer has the field for entering that value. Setting to `1` sets it to `count` (albeit unsafely since the ids aren't stable). Probably need to either re-add that form control so it can be set by the user, or else make that field nullable and pass null here instead.

Note to self:
- Check both edit and create technique pages
- Check `ObservationCountDataGridEditCell` as it is also setting this value to `1`

Reply:
Fixed. Added proper form control to populate this field.